packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -72,6 +72,8 @@ export type ClinicalDataBinFilter = {
 
         'end': number
 
+        'showNA': boolean
```

Comment:
This was manually copied over from `packages/cbioportal-ts-api-client/dist/generated/CBioPortalAPIInternal.d.ts`

Reply:
I think this is an auto-generated file, it should not be updated